extensions/wikia/Email/Controller/PasswordResetLinkController.class.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Fatal;
+
+/**
+ * Class PasswordResetLinkController
+ *
+ * @requestParam int targetUserId : The user id to send the password reset link email to
+ * @requestParam string token : The token by which a user will be identified
+ *
+ * @package      Email\Controller
+ */
+class PasswordResetLinkController extends EmailController {
+
+	protected $token;
+	protected $returnUrl;
+	const MAX_LINK_LENGTH = 40;
+
+	const URL_NO_RETURN = 'http://dummy-address/%s/%s';
+	const URL_WITH_RETURN = 'http://dummy-address/%s/%s?return_url=%s';
```

Comment:
I think so, ideally at least, since it will make QA a lot easier, so you don't have to manually change the URL when testing.